mmuav:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  joint1_left_controller:
    type: effort_controllers/JointPositionController
    joint: joint_q1_left
    pid: {p: 30.0, i: 0.005, d: 0.0, i_clamp_max: 100.0, i_clamp_min: -100.0}
  joint2_left_controller:
    type: effort_controllers/JointPositionController
    joint: joint_q2_left
    pid: {p: 30.0, i: 0.005, d: 0.0, i_clamp_max: 100.0, i_clamp_min: -100.0}
  joint1_right_controller:
    type: effort_controllers/JointPositionController
    joint: joint_q1_right
    pid: {p: 30.0, i: 0.005, d: 0.0, i_clamp_max: 100.0, i_clamp_min: -100.0}
  joint2_right_controller:
    type: effort_controllers/JointPositionController
    joint: joint_q2_right
    pid: {p: 30.0, i: 0.005, d: 0.0, i_clamp_max: 100.0, i_clamp_min: -100.0}
